fips202/fips202.h

```diff
@@ -55,6 +55,8 @@ void shake256_inc_finalize(shake256incctx *state);
  */
 void shake256_inc_squeeze(uint8_t *output, size_t outlen, shake256incctx *state);
 
+void sha3_256_inc_finalize(shake256incctx *state);
+
```

Comment:
This does not make any sense. If it's a shake256incctx, you can't create a SHA3_256 hash from that.
Also I'd prefer if you don't extend the fips202 api for the tests. 

Just use `shake256_inc_squeeze` with a 32-byte output.

Reply:
I don't understand why I can replace finalize by squeeze, I thought it's more similar to absorb ?
But I moved the sha3 finalize function into the nistkat.c file avoid changing the interface of fips202, let me know if this is ok for you

Reply:
You will need both a finalize and a squeeze. For SHA3 it is only one call since the output length is fixed.